### PR TITLE
add information about extra fields in m:n section

### PR DESCRIPTION
**Pull request by divyenduz**

### CODE REVIEW

File: schema/Readme.md [hunk 1]
```diff
@@ -507,7 +507,7 @@ we'll use `primary key(blog, writer)` to ensure that there can't be no more than
 For implicit many-to-many relations, you **must** include both `Blog.authors` and `Writer.blogs`. If one of these fields is missing, Prisma will assume it's a
 **One-to-Many (1:N)** relationship.
 
-##### Explicit Many-to-Many (M:N) Relationships
+##### Explicit Many-To-Many (M:N) Relationships With Or Without Extra Columns
```
Comment (janpio) [inline schema/Readme.md]: inconsistent with previous headline

Comment (divyenduz) [reply schema/Readme.md]: What do you mean?

Comment (janpio) [reply schema/Readme.md]: image

Comment (divyenduz) [reply schema/Readme.md]: Do you have a suggestion for improving this? Extra columns doesn't apply to implicit m:n relation in my understanding.

Comment (janpio) [reply schema/Readme.md]: The previous headline has "Many-to-Many".
You changed this one to be "Many-To-Many".
Not the lowercase vs. the Uppercase `t`.

Comment (divyenduz) [reply schema/Readme.md]: aha! okay, that is descriptive, let me make all titles to be in title case! (as per VSCode) unless we have a special way to represent titles?

Comment (janpio) [reply schema/Readme.md]: The existing way to write Many-to-Many was perfectly fine.

Comment (divyenduz) [reply schema/Readme.md]: Fixed

**Comment by divyenduz** [comment]
@janpio this good to merge now?

**Pull request merged**